src/main/java/com/puppycrawl/tools/checkstyle/xpath/AbstractNode.java

```diff
@@ -157,6 +157,15 @@ public final TreeInfo getTreeInfo() {
         return treeInfo;
     }
 
+    /**
+     * Returns string value.
+     * @return string value
+     */
+    @Override
+    public String getStringValue() {
+        throw throwUnsupportedOperationException();
```

Comment:
`throw throw`

method should be named as "createUnsupportedOperationException"

Reply:
done